apps/ui/src/components/ProposalsListItemHeading.vue

```diff
@@ -136,7 +136,12 @@ const totalProgress = computed(() => quorumProgress(props.proposal));
         type="button"
         class="text-skin-text"
         @click="modalOpenTimeline = true"
-        v-text="_rt(getTsFromCurrent(proposal.network, proposal.max_end))"
+        v-text="
+          proposal.state === 'pending'
+            ? 'Start in ' +
```

Comment:
It should be start**s** and here we use only lowercase
```suggestion
            ? 'starts in ' +
```

Reply:
It uses uppercase here: https://github.com/snapshot-labs/sx-monorepo/pull/1326/files#diff-96ddc96602d1663b24a80682c60369be4c86377460e31d6eaf92c6a8b78c8055R129